packages/api/AlvTimeWebApi/Controllers/Admin/TimeEntriesAdminController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using AlvTime.Business.TimeRegistration;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System;
+using AlvTimeWebApi.Authentication;
+using System.Linq;
+using AlvTimeWebApi.Utils;
+
+namespace AlvTimeWebApi.Controllers.Admin
+{
+    [Route("api/admin")]
+    [ApiController]
+    [AuthorizeOrakelet]
+    public class TimeEntriesAdminController : Controller
+    {
+        private readonly ITimeRegistrationStorage _storage;
+
+        public TimeEntriesAdminController(ITimeRegistrationStorage storage)
+        {
+            _storage = storage;
+        }
+
+        [HttpGet("TimeEntries")]
+        public async Task<ActionResult<IEnumerable<TimeEntryEmployeeResponseDto>>> GetTimeEntriesForEmployees([FromQuery] int[] employeeIds, [FromQuery] int[] taskIds, DateTime fromDate, DateTime toDate)
+        {
+            try
+            {
+                return Ok((await _storage.GetTimeEntriesForEmployees(new MultipleTimeEntriesQuerySearch
+                {
+                    EmployeeIds = employeeIds,
+                    FromDateInclusive = fromDate,
+                    ToDateInclusive = toDate,
+                    TaskIds = taskIds
+                }))
+                    .Select(timeEntry => new
+                    {
+                        User = timeEntry.User,
+                        EmployeeId = timeEntry.EmployeeId,
+                        Date = timeEntry.Date.ToDateOnly(),
+                        Value = timeEntry.Value,
+                        TaskId = timeEntry.TaskId,
+                        ProjectId = timeEntry.ProjectId,
+                    }));
+            }
+            catch (Exception e)
```

Comment:
Du kan sikkert fjerne try/catch her. Vi har en ErrorHandlingMiddleware som fanger exceptions. BadRequest pleier også gjerne å bli brukt for valideringsfeil, mens denne vil nok for det meste fange databasefeil som kanskje heller er 500-feil.